configs/config-depl.json

```diff
@@ -81,6 +81,7 @@
       "id": "iudx.aaa.server.apd.ApdVerticle",
       "verticleInstances": 1,
       "required":["postgresOptions", "commonOptions"],
+      "webClientTimeoutMs": 4000,
```

Comment:
The integration tests for the PR pipeline may fail due to this value not being there. @karun-singh could you please add it to the integration config?

Reply:
Added

Reply:
Thanks!